aten/src/ATen/native/transformers/cuda/sdp_utils.cpp

```diff
@@ -173,6 +173,23 @@ bool check_flash_attention_hardware_support(sdp_params const& params, bool debug
   using sm80 = SMVersion<8, 0>;
   using sm90 = SMVersion<9, 0>;
   auto dprops = at::cuda::getCurrentDeviceProperties();
+#if USE_ROCM
+  constexpr std::string_view mi200 = "gfx90a:sramecc+:xnack-";
```

Comment:
Can you please explain what is this line? And you define the same string constant later in tests, perhaps it could be avoided?